### ISSUE TITLE
CSS `min()` function breaks syntax highlighting

### ISSUE THREAD

**Issue by stefanprobst** [microsoft/vscode#134243]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.60.2 (with `--disable-extensions`)
- OS Version: Ubuntu 20.04

Using CSS `min()` or `clamp()` functions seems to break syntax highlighting.

Steps to Reproduce:

1. Copy-paste the following css rule, and observe the different highlighting for the first `1fr` and the second `1fr` value:

```css
.test {
  grid-template-columns: 1fr min(1024px, 100%) 1fr;
}
```
![Screenshot_20210930_222956](https://user-images.githubusercontent.com/20753323/135526081-ae327b2e-e1e9-415f-9441-3f2f974e0c44.png)

![Screenshot_20210930_223201](https://user-images.githubusercontent.com/20753323/135526186-f658f8a7-21cd-43e1-89a1-adb12327c76b.png)

![Screenshot_20210930_223309](https://user-images.githubusercontent.com/20753323/135526320-332d0afe-223b-44d9-80c3-321b92a50fd4.png)

**Comment by stefanprobst** [comment]
looking a bit into this, is it correct that this would need to be fixed in the upstream `language-css` repo. if yes, could you clarify if [this one](https://github.com/atom/language-css) is the right one, or [this fork](https://github.com/octref/language-css), which has also been linked to in issue comments. thanks

**Comment by alexr00** [comment]
@stefanprobst you are correct that atom/language-css is the upstream repo for this issue! I have copied the issue there. https://github.com/atom/language-css/issues/186
